sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/EnvironmentResourceTest.java

```diff
@@ -39,12 +39,36 @@ void resourceFromConfig() {
                     "service.name=myService,appName=MyApp")));
 
     assertThat(attributes)
-        .isEqualTo(
-            Attributes.of(
-                ResourceAttributes.SERVICE_NAME,
-                "myService",
-                AttributeKey.stringKey("appName"),
-                "MyApp"));
+        .hasSize(2)
```

Comment:
This pattern is ok and supported by assertj for Maps so I think it's good to add them. But just wondering whether you prefer it to `containsOnly(entry(...), attributeEntry(...))` (no hasSize) - maybe because of the two different entry methods?

Reply:
It's a good question. The current Map.Entry style for only is pretty ugly. We can always add more in the future.

Reply:
To clarify: I prefer this personally, since I think Map.Entry is not a very pretty interface to use for this, IMO. (It works for those who prefer that way of doing it, of course).